[MFLOG|0|2022-10-21T20:47:01.285947Z|runtime|99056854-813a-4f74-938c-a4eea5d3f275]The command is: 
[MFLOG|0|2022-10-21T20:47:05.620771Z|runtime|d1aa5642-d91a-4597-a8d5-3f6993076c8f] 
[MFLOG|0|2022-10-21T20:47:05.621042Z|runtime|d4589707-59fe-4df1-9b2e-fc786cb96b26]        docker run         --rm         -i         -d         --name python-19407         --mount type=bind,source=/home/abd/Desktop/Work/Metaflow/CMS-Analysis,target=/home/abd/Desktop/Work/Metaflow/CMS-Analysis         gitlab-registry.cern.ch/cms-cloud/python-vnc &&         docker start python-19407 &&         docker exec -it python-19407 bash /home/abd/Desktop/Work/Metaflow/CMS-Analysis/shell-scripts/Bash/flattentrees-bash.sh /home/abd/Desktop/Work/Metaflow/CMS-Analysis/vol 19407
[MFLOG|0|2022-10-21T20:47:05.621371Z|runtime|a2e04a2c-3875-4300-bed0-b60fe57701f0]        
[MFLOG|0|2022-10-21T20:47:05.621548Z|runtime|10ede589-a77c-479e-b2eb-29612f8b8ee0]The output is: 
[MFLOG|0|2022-10-21T20:47:05.621700Z|runtime|d2172381-38a3-4584-9868-bc6686d7f00a] a93189f692c7aced65a0be65f8b78ab8669c1e50ea3b2fcb5bee1e9bd809f0b6
[MFLOG|0|2022-10-21T20:47:05.621830Z|runtime|44643342-a843-496e-9bed-ffa85bf7999c]python-19407
[MFLOG|0|2022-10-21T20:47:05.621957Z|runtime|7ffcf54b-a49d-42de-93d6-0c68e85917e7]
[MFLOG|0|2022-10-21T20:47:05.622087Z|runtime|a9b825b0-a581-48bd-8eb4-57d984d9ba4f]Return Code: 1
[MFLOG|0|2022-10-21T20:47:05.622212Z|runtime|45d2886a-34c9-4597-a9b5-1394a7762c4a]The error is: 
[MFLOG|0|2022-10-21T20:47:05.622333Z|runtime|15701bfa-c972-4a12-a9ed-3d38a2819359] the input device is not a TTY
[MFLOG|0|2022-10-21T20:47:05.622466Z|runtime|06bd329c-546e-4591-b431-a008af6fe29d]
